packages/mdx/lib/plugin/recma-jsx-rewrite.js

```diff
@@ -195,21 +195,24 @@ export function recmaJsxRewrite(options = {}) {
               fnScope.tags.push(id)
             }
 
-            let componentName = toJsxIdOrMemberExpression(['_components', id])
+            /** @type {(string | number)[]} */
```

Comment:
```suggestion
            /** @type {Array<string | number>} */
```

Rest of the code uses `Array<_>`, so for consistency.
I prefer it because a) it’s more in line with other types over something else, b) it’s the way I’d read something, c) `[]` is also used for tuples